pom.xml

```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.aerospike</groupId>
+    <artifactId>spring-boot-data-aerospike</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>Autoconfiguration for Spring Data Aerospike</name>
+
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>spring-boot-autoconfigure-data-aerospike</module>
+        <module>spring-boot-starter-data-aerospike</module>
+    </modules>
+
+    <properties>
+        <java.version>1.8</java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <spring-data-aerospike.version>2.2.0.RELEASE</spring-data-aerospike.version>
+
+        <spring-boot.version>2.2.0.RELEASE</spring-boot.version>
+
+        <aerospike-reactor-client.version>4.4.6</aerospike-reactor-client.version>
+
+        <!-- Plugins -->
+        <jacoco-plugin.version>0.8.5</jacoco-plugin.version>
+
+        <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
```

Comment:
Probably we could skip all these maven plugins configuration if we used spring-boot-parent

Reply:
Plugins configuration removed.